LICENSE

```diff
@@ -1,201 +1,201 @@
-                                 Apache License
```

Comment:
Looks like `editorconfig` got a hold of this file

Reply:
Ugh, I tried to fix this file but apparently I failed :(